computervision/src/main/java/ComputerVision.kt

```diff
@@ -27,7 +27,7 @@ sealed class Vision : SensorReading {
     class Scenery(val scene: String) : Vision()
     class Facial (val prettyFace: String) : Vision()
     class Emotion (val emotion: String) : Vision()
-    class ObjectDetection (val objects: List<Any>): Vision()
+    class ObjectDetection (val objects: List<String>): Vision()
```

Comment:
Each detected object is represented by a `label` and a `pos` values. not a single string.
Maybe write a new class `DetectedObject(val label: String, val position: String)`.
No this line becomes:
``` kotlin
  class ObjectDetection (val objects: List<DetectedObject>): Vision()
```

Reply:
Okay, I got that we need only the two values as literal Strings, but this class thing is not clear for me.
I thought it could be something like this:
-------------------------------------------------------------------------- 
OBJECTS -> Vision.ObjectDetection(listOf(raw.getJSONArray("value").getJSONObject(/*some index*/).getString("label"), raw.getJSONArray("value").getJSONObject(/*some index*/).getString("pos")))
--------------------------------------------------------------------------
but this needs to be iterated

Reply:
Really people?, this is not supposed to be bold, I am sorry 🙇‍♀

Reply:
**HOW BOLD OF YOU!**
---
Well you're right, but not 100% correct; what happens if we wanted to add one more feature (of type `Int`) to the detected object? then it's not possible to put all the elements in a `List<String>`. In OOP land, we use objects for almost everything, and we combine objects to form a new one, that's why need to __encapsulate__ (or map) the two strings of each `JSONObject` to a Kotlin object.

so that
``` javascript
// JSON
[{'label': 'dog', 'pos': 'right'}, {'label': 'cat', 'pos': 'left'}]
```
becomes:
``` kotlin
// Kotlin
listOf(DetectedObject("dog", "right"), DetectedObject("cat", "left"))
```